### DISCUSSION TITLE
PR merge 방식 변경

### DISCUSSION THREAD

**Discussion by devitssu** [one-day-one-meal/one-hour-one-meal#45] [Ideas]
현재 PR후 브랜치 merge할 때 `creat a merging commit` 방식을 쓰고 있습니다. 작업한 모든 커밋을 볼 수 있다는게 장점이라고 생각했는데 코드리뷰를 진행하다 보니 사소한 수정을 위한 커밋이 너무 많아 질 것 같아서 `sqush and merge` 방식으로 바꾸는게 나은지 고민됩니다.
어떤 방식이 더 좋다고 생각하시나요?

**Comment by mobzzzzz** [comment]
보통 커밋 로그가 많아서 지저분하다고 느껴지면 Squash merge를 권장하는 걸로 알고 있는데 저는 좀 익숙한 방식이라 바꾸면 편하고 좋긴 합니다!
